includes/html/forms/delete-alert-rule.inc.php

```diff
@@ -31,6 +31,10 @@
         $alert_msg_prefix .= ' id ' . $vars['alert_id'];
     }
     if (dbDelete('alert_rules', '`id` =  ?', [$vars['alert_id']])) {
+        // @phpstan-ignore-next-line
```

Comment:
db*() functions are deprecated and will be deleted.  Let's try not to add new lines using them. :)

Alert::where('rule_id', $vars['alert_id'])->delete();
AlertLog::where('rule_id', $vars['alert_id'])->delete();

Reply:
Agree, as AlertTemplateMap has a class too, I changed it too.